lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -7,6 +7,7 @@ module Engine
     module Analyzers
       module Ruby
         class Main
+          DEFAULT_PATHS = ["**/*.rb", "**/*.rake", "Rakefile"]
```

Comment:
For Ruby, you can find Rubocop's defaults here:

https://github.com/bbatsov/rubocop/blob/aa7d73992fcd1abbf3b57e56e6d9888a8e45d682/config/default.yml